src/MosaicViews.jl

```diff
@@ -324,8 +330,7 @@ function mosaicview(As::AbstractVector{T};
                     kwargs...) where {T <: AbstractArray}
     length(As) == 0 && throw(ArgumentError("The given vector should not be empty"))
     N = ndims(first(As))
-    2 <= N || throw(ArgumentError("The given array must have dimensionality of N=2 or higher"))
-    mosaicview(_padded_cat(As; center=center, fillvalue=fillvalue, dims=N+1);
+    mosaicview(_padded_cat(As; center=center, fillvalue=fillvalue, dims=max(3, N+1));
```

Comment:
In order to make those keyword arguments work as expected, `vectors` are concatenated in the third dimension.